### PR TITLE
Filestore optim v3

### PR DESCRIPTION
**Pull request by regit**
Update of #2738 addressing comments

PR builds:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/281
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/65

### CODE REVIEW

File: src/log-filestore.c [hunk 1]
```diff
@@ -64,12 +64,21 @@
 
 static char g_logfile_base_dir[PATH_MAX] = "/tmp";
 
+SC_ATOMIC_DECLARE(uint32_t, filestore_open_file_cnt);  /**< Atomic counter, to link relative event */
```
Comment (inliniac) [inline src/log-filestore.c]: ehh

File: src/log-filestore.c [hunk 2]
```diff
@@ -502,9 +574,31 @@ static OutputCtx *LogFilestoreLogInitCtx(ConfNode *conf)
         }
     }
 
+    const char *file_count_str = ConfNodeLookupChildValue(conf, "max-open-files");
+    if (file_count_str != NULL && strcmp(file_count_str, "no")) {
```
Comment (inliniac) [inline src/log-filestore.c]: hm?

**Comment by inliniac** [comment]
Did you push the correct branch? Multiple comments from #2738 not addressed.

File: src/log-filestore.c (hunk 2 above)
Comment (regit) [inline src/log-filestore.c]: not the good one indeed.

**Comment by regit** [comment]
Stupid mistake, lost of few things after rebase.

**Pull request closed without merging**